src/test/java/bio/terra/service/snapshot/SnapshotMinimalConnectedTest.java

```diff
@@ -138,6 +140,15 @@ public void testMinimal() throws Exception {
     assertThat("to table is right", relationshipModel.getTo().getTable(), equalTo("sample"));
     assertThat(
         "to column is right", relationshipModel.getTo().getColumn(), equalTo("participant_id"));
+
+    SnapshotPreviewModel snapshotPreviewModel =
+        SnapshotConnectedTestUtils.getTablePreview(
+            connectedOperations, summaryModel.getId(), "participant", 10, 0);
+
+    assertThat(
+        "participant has the correct age",
+        ((LinkedHashMap) snapshotPreviewModel.getResult().get(0)).get("age"),
+        equalTo("23"));
```

Comment:
Could you also add an assert that the result size equals 10?

Reply:
The table has <10 rows, so it will return that subset. I think that should be the desired behavior, since it may not be possible to know ahead of time how many rows a table has.

Reply:
Additionally, there is code that checks the returned number of rows already.